prompting/tasks/inference.py

```diff
@@ -33,7 +37,16 @@ class InferenceTask(BaseTextTask):
     query: str | None = None
     reference: str | None = None
     llm_model: ModelConfig | None = None
-    seed: int = random.randint(0, 1_000_000)
+    llm_model_id: ModelConfig | None = random.choice(ModelZoo.models_configs).llm_model_id
+    seed: int = Field(default_factory=lambda: random.randint(0, 1_000_000))
+
+    @model_validator(mode="after")
+    def random_llm_model_id(self):
+        if np.random.rand() < 0.2:
+            self.llm_model_id = None
+        else:
+            self.llm_model = ModelZoo.get_model_by_id(self.llm_model_id)
```

Comment:
Make sure to check the run. The same PR to pre-staging was causing OOMs

Reply:
This should not be the cause of OOM I think, it would be the change to the model_config I believe. I don't see how this can lead to OOM since it's not using any memory, right?